adaptation/commons.py

```diff
@@ -52,6 +53,57 @@ def notify(*args, **kwargs):
 
 
 @app.task()
+def image_processing(src, dest):
+    print "(------------"
+    random_uuid = uuid.uuid4().hex
+    context={"original_file": src, "folder_out": config["folder_out"] + dest + "/", "id": random_uuid}
+    
+    if not os.path.exists(context['folder_out']):
+        os.makedirs(context['folder_out'])
+    
+    ext = src.split('.');
+    ext = ext[len(ext)-1]
+
+    media_info = MediaInfo.parse(context["original_file"])
+
+    for track in media_info.tracks:
+	width = track.width
+	print width
+
+    extralarge = 1382
+    large = 992
+    medium = 768
+    small = 480
+
+    ffargsoriginal = "ffmpeg -i " + src + " -vf scale=" + str(width) +":-1 " + context["folder_out"] + "original.jpg"
```

Comment:
if you download the "original file" on snapmail you download the real original or this ?